py/usageaccountant/datadog_fetcher.py

```diff
@@ -294,15 +294,18 @@ def process_series_data(
     for series in series_list:
         app_feature = series["scope_dict"]["app_feature"]
         for point in series["pointlist"]:
-            record_list.append(
-                UsageAccumulatorRecord(
+            try:
+                record = UsageAccumulatorRecord(
                     resource_id=shared_resource_id,
                     app_feature=app_feature,
                     # point[0] is the timestamp
                     amount=int(point[1]),
                     usage_type=parsed_unit,
                 )
-            )
+            except TypeError as e:
+                logger.warning(e, exc_info=True)
+                continue
+            record_list.append(record)
```

Comment:
Please do not use a try catch block to decide what to do when finding a null value. 
```
if point is not None:
    .....
```